_layouts/default.html

```diff
@@ -6,7 +6,27 @@
 {%- include hero.html -%}
 {%- endif -%}
 
-{{ content }}
+  <!-- Comment to remove TOC from a page -->
+{% if page.show_pagenav  %}
+<div class="row" style="margin-left:0; margin-right:0;">
+  <div class="col-md-2 col-sm-12">
+  </div>
+  <div class="col-md-8 col-sm-12">
+    <div class="mx-2">
+      {% include anchor_headings.html html=content anchorBody="#" %}
+    </div>
+  </div>
+  <div class="col-md-2 d-md-block d-none">
```

Comment:
We need to make sure our TOC is getting more width for medium screens, with only md-2 it gets unsightly if you make things narrower:
<img width="788" alt="image" src="https://user-images.githubusercontent.com/104291808/232561091-1ed592de-08c6-4db8-9886-47f8030bcdab.png">

You can see what I settled on for the Blog TOC in the blog.html file.

Reply:
Lets debug together in our meeting later ... using md-3 just didnt look so good.. there was LOTS of additional space.